static/styles/components.css

```diff
@@ -52,6 +52,58 @@
     a.no-style {
         color: inherit;
     }
+
+    label.checkbox_todo {
+        padding: 0;
+        display: inline-block;
+        position: relative;
+        vertical-align: top;
+
+        input[type="checkbox"] {
+            position: absolute;
+            clip: rect(0, 0, 0, 0);
+
+            ~ span {
+                display: inline-block;
+                vertical-align: middle;
+                position: relative;
+                top: -2px;
+
+                padding: 2px;
+                margin: 0 5px 0 0;
+                height: 12px;
+                width: 12px;
+
+                font-weight: 300;
+                line-height: 0.8;
+                font-size: 1.3rem;
+                text-align: center;
+                border: 2px solid hsl(156, 28%, 70%);
+
+                border-radius: 4px;
+
+                cursor: pointer;
+            }
+
+            &:checked ~ span {
+                background-image: url("../images/checkbox-white.svg");
+                background-size: 75%;
+                background-position: 50% 50%;
+                background-repeat: no-repeat;
+                background-color: hsl(156, 41%, 40%);
+                border: 2px solid hsl(156, 41%, 40%);
+            }
+
+            &:disabled ~ span {
+                opacity: 0.5;
+                cursor: not-allowed;
+            }
+
+            &:hover ~ span {
+                border-color: hsl(156, 41%, 40%);
+            }
+        }
```

Comment:
A lot of the rules between `label.checkbox` and `label.checkbox_todo` are duplicate, so can we just apply `checkbox` and just override the rules we want to change with `checkbox_todo`?